app/models/list.rb

```diff
@@ -39,4 +39,12 @@ def add_item params
   def is_shared_with?(u)
     user != u
   end
+
+  def shared_list_owner
+    user.first_name + " " + user.last_name
+  end
+
+  def number_shared_with
```

Comment:
Nit: the name feels heavy.  What about `share_count` ?

Reply:
Commit incoming.

Reply:
Fixed